### ISSUE TITLE
iPad connection to Mac running dnscrypt-proxy

### ISSUE THREAD

**Issue by tthkbw** [DNSCrypt/dnscrypt-proxy#360]
I have set up dnscrypt-proxy to run on my iMac (10.13.4) using Cloudflare servers. I have installed dnscrypt-proxy as a service and everything works as expected. 

However, when I plug in my iPad Pro 10.5 (iOS 11.3) to the iMac, the iPad loses internet connectivity. On the iPad, the wireless indicator on the status bar disappears. Settings show that the iPad is still connected to WIFI, but Safari, etc no long have internet connectivity. If I run Airport Utility on the iPad while connected to the iMac, it finds and communicates with an Apple Time Capsule which is on the network, so wifi is working, it is just internet connectivity that is gone. Also, the usb connection between the iMac and the iPad works properly.

Can anyone explain this and/or suggest a fix?

**Comment by publicarray** [comment]
Huh? I've never heard of that before. I recommend to look at Apple Support, Alternatively you can reset the network settings in the iPad: `Settings>General>Reset (it's at the very bottom)> Reset Network Settings` note this will wipe any remembered WiFi passwords so you need to re-enter them.

**Comment by publicarray** [comment]
Typically the Router will have dhcp which will give any connected device the recommended DNS servers to connect to. By using dnscrypt-proxy your iMac is using it's own DNS settings which does not impact the rest of the network. When the iPad is connected to the iMac, the iPad would still use it's own DNS settings and wifi, so I don't know how dnscrypt-proxy would have an impact.

**Comment by publicarray** [comment]
On https://discussions.apple.com/thread/2746307 there is a mention of Internet sharing, make sure that it's off:
![screen shot](https://user-images.githubusercontent.com/5497998/38773282-9eb33f0e-408b-11e8-88e7-e14d9112128f.png)

**Comment by tthkbw** [comment]
publicarray:

Thank you! I knew there was something I had changed very recently that had to do with iPad and iMac connecting, but couldn't remember it and searched without coming across it.

In fact, I had turned on "Content Sharing" just after loading 10.13.4. Turning on content sharing also turned on internet sharing--there is a check box under content sharing that, I believe, was checked by default. Unchecking that box and disabling internet sharing solved the problem.

Weird.
